components/script/dom/document.rs

```diff
@@ -2222,12 +2224,12 @@ impl Document {
             has_browsing_context: has_browsing_context == HasBrowsingContext::Yes,
             implementation: Default::default(),
             content_type: match content_type {
-                Some(string) => string,
-                None => DOMString::from(match is_html_document {
+                Some(string) => string.parse().unwrap(),
```

Comment:
`unwrap()` here will panic if we ever get a content type that isn't a valid mime type. Let's use `unwrap_or` and pass a default value of text/html.

Reply:
Perfect, I will do that.

Reply:
After updating params of new_inherited, we can directly assign the value of string(like it was previously) without even using unwrap.

Reply:
Oh, right. Even better!